src/ai/susi/server/api/aaa/GetAAAFiles.java

```diff
@@ -0,0 +1,43 @@
+package ai.susi.server.api.aaa;
+
+import ai.susi.DAO;
+import ai.susi.json.JsonObjectWithDefault;
+import ai.susi.server.*;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import javax.servlet.http.HttpServletResponse;
+import java.io.File;
+
+/**
+ * Created by dravit on 14/6/17.
+ * Servlet to get list of all the files present in data/settings directory
+ * test locally at http://127.0.0.1:4000/aaa/getAllFiles
+ */
+public class GetAAAFiles extends AbstractAPIHandler implements APIHandler{
+    @Override
```

Comment:
This lists settings. The files inside the settings folder are not al about AAA. Please name it ListSettingsService. All Classes which extends AbstractAPIHandler shall be named a "service"